### PR TITLE
explain than a css theme is required

### LINKED ISSUE #242: how to set Syntax highlighting for code with remark-prismjs ?
## Description
Hello, i tried to install remark-prismjs to highlight my code block but my code is still not highlighted. https://yineo.fr/post/animer-vos-transitions-de-page-facilement-avec-nuxt-js-et-vue-js
Shoud I include manually a js file ?

Here is my gridsome.config.js

```js
module.exports = {
  plugins: [
    {
      use: '@gridsome/source-filesystem',
      options: {
        path: 'content/posts/*.md',
        typeName: 'BlogPost',
        route: '/post/:slug',
        refs: {
          tags: 'Tag',
        },
        remark: {
          plugins: [
            '@gridsome/remark-prismjs'
          ]
        }
      }
    },
    {
      use: '@gridsome/source-filesystem',
      options: {
        path: 'content/tags/*.md',
        typeName: 'Tag',
        route: '/tag/:id'
      }
    },
    {
      use: '@gridsome/source-filesystem',
      options: {
        path: 'content/blocks/*.md',
        typeName: 'Block'
      }
    }
  ]
}
```

and my `package.json`

```js
{
//
  "dependencies": {
    "@gridsome/remark-prismjs": "^0.0.4",
    "@gridsome/source-filesystem": "0.3.0",
    "@gridsome/transformer-remark": "0.2.0",
    "axios": "^0.18.0",
    "gridsome": "^0.5.0",
    "striptags": "^3.1.1"
  },
//
}
```

### Environment

<pre><code>
Libs:
- gridsome version: 0.5.0
- @gridsome/cli version: 0.0.7

Browser:
- [X] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

### PR DESCRIPTION
**Pull request by yann-yinn**
I struggled here about that : https://github.com/gridsome/gridsome/issues/242 

---

Closes #242.

### CODE REVIEW

**Comment by hjvedvik** [comment]
Yes, it should be mentioned in the readme :) Can you add a simple `main.js` example in the «Usage» section instead? Something like:

```js
import 'prismjs/themes/prism.css'

export default function (Vue) {
  // ...
}
```

**Comment by yann-yinn** [comment]
done 👍

File: packages/remark-prismjs/README.md [hunk 1]
```diff
@@ -8,7 +8,17 @@
 
 ## Usage
 
-Add syntax highlighter to a single markdown source:
+In your `main.js` file, import a Prism css theme
```
Comment (hjvedvik) [inline packages/remark-prismjs/README.md]:
```suggestion
In your `main.js` file, import a Prism CSS theme:
```

**Comment by hjvedvik** [comment]
Thanks :)

**Pull request merged**